source/user_guide/concepts.rst

```diff
@@ -0,0 +1,115 @@
+a range of zero to infinity. A conviction value of one is average, as the actual information is equal to the expected information. Convictions less than one indicate higher surprisal, as the actual information
+iss less than the expected information, while convictions greater than one indicate lower surprisal, since the actual information is higher than the expected information. 
+
+Actual information is the amount of information that a data point actually adds when it is evaluated. Expected information is what the value of a new data point should be on average, given
+all knowledge of the data around it. For example, say you are again trying to make heart disease predictions, given historical heart disease data and a new patient's data is being analyzed.
+This patient has features very similar to historical data with positive prediction status. Thus, its expected heart disease prediction value is positive. Its actual information, though, is the true information
+about whether the patient has heart disease. If the patient does, in fact, have heart disease, then its conviction value would be around one, as its actual information is similar to its expected information.
+However, if the patient actually does not have heart disease, its actual value is not what was expected, so the conviction would be low, indicating a surprising outcome. While this is an example, the concept of surprisal,
+quantified by conviction, is prevalent throughout many of Howso's analysis capabilities and can be adapted for a variety of tools and use cases.
+
+Basic Workflows
+^^^^^^^^^^^^^^^
+
+There are three main components of a Howso Engine workflow: building a `Trainee`, `analyzing` the data, and `reacting` to new data.
+
+Trainee
+-------
+
+A `Trainee` is a collection of data, metadata, and hyperparameters upon which the IBL algorithm acts. This is analogous to a model in traditional machine learning settings, but is 
+different in that it is the *actual data*, stored in memory, and not an abstract representation of the data.
+
+Analyze
+-------
+
+Once a `Trainee` is built, i.e., the data is stored, it is `analyzed` to understand the relationships between the individual data points. Here, the internal parameters describing these
+relationships are tuned to improve performance and the accuracy of predictions and metrics. 
+
+React
+-----
+
+Finally, after building the `Trainee` and `analyzing`, Howso Engine can be used for a variety of data-driven analysis applications. Often, these applications involve
+`reacting` to new data, which involves exposing the Trainee to new information and using the underlying IBL algorithm to make a prediction about that new information.
+This is equivalent to predicting or labeling in many traditional machine learning workflows, although `reacting` can be used for a variety of analyses, in addition to supervised
```

Comment:
React into trainee is a important part of workflow too, you don't need to specifically call it out but I would say something along the lines of "React can also be used to analyze data already trained into the Trainee etc..".